src/locales/en-US/admin.ftl

```diff
@@ -1235,12 +1233,18 @@ community-userStatus-changeButton =
 community-userStatus-premodUser = Always pre-moderate
 community-userStatus-removePremod = Remove pre-moderate
 
-community-banModal-areYouSure = Are you sure you want to ban <username></username>?
-community-banModal-consequence =
+community-banModal-allSites-title = Are you sure you want to ban <username></username>?
+community-banModal-specificSites-title = Are you sure you want to manage ban status of <username></username>?
```

Comment:
Maybe adding a `the` to make this `...to manage the ban status of...` would flow better?